### PR TITLE
feat: add nodes script

### PR DESCRIPTION
**Pull request by frdomovic**
# feat: add nodes script

## Summary
Shell script for starting nodes and has 2 options:
1. delete-and-start - deletes the data folder and recreates all the files : coordinator, node1, node2
2. start - starts the nodes + coordinator with existing files in data folder.

Video: 

https://github.com/calimero-network/core/assets/93442516/6bae697f-2715-40b6-9efe-a2b61166b34f

## Summary by CodeRabbit

- **New Features**
    - Added functionality to manage the initialization and running of multiple nodes using Cargo.
    - Options to delete and start nodes or simply start them were included.

### CODE REVIEW

File: start-nodes.sh [hunk 1]
```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+show_usage() {
+    echo "Usage: $0 delete-and-start || start"
```
Comment (its-saeed) [inline start-nodes.sh:4]: Shouldn't we send this error to STDERR? 🤔

File: start-nodes.sh [hunk 2]
```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
```
Comment (its-saeed) [inline start-nodes.sh:1]: what about having `set -e` at the beginning?

**Comment by xilosada** [comment]
@frdomovic I can see this as functionality provided by dev-tool as. It can be added to our app template as developer dependencies and configured during the installation. IMO this should not be in this repo. Cc: @MatejVukosav

**Comment by xilosada** [comment]
Also I think you don't need osascript for any of this

**Pull request closed without merging**